contrib/clojure-package/README.md

```diff
@@ -14,43 +14,41 @@ For a **video introduction**, see [Clojure MXNet with Carin Meier - Clojure Virt
 
 ## Current State and Plans
 
-The Clojure package is nearing the end of its first development milestone which is to achieve a close parity with the Scala package.
-
 Help is needed testing and generally making the package better. A list of the pacakge status and contribution needs can be found here [Clojure Package Contribution Needs](https://cwiki.apache.org/confluence/display/MXNET/Clojure+Package+Contribution+Needs). Please get involved :)
 
-Testing instructions can be found in the testing.md.
 
 ## Getting Started
 
+The Clojure MXNet framework consists of a core C library, a Scala Api that talks to it through JNI bindings, and finally a Clojure wrapper around the Scala api.
+
+Since there is a native code involved in the framework, what OS you are running matters.
+
 The following systems are supported:
 
 - OSX cpu
 - Linux cpu
 - Linux gpu
 
-There are two ways of getting going. The first way is the easiest and that is to use the pre-built jars from Maven. The second way is to build from source. In both cases, you will need to load the prereqs and dependencies, (like opencv).
-
-
-
-### Prerequisites
+There are three ways of getting started:
 
+* Use the Clojure jars with the native dependencies baked in. This the easiest way to get going.
+* Checkout the MXNet project from a release tag using the Scala jars with native deps. This is also a pretty easy way to get started.
+* Build from the MXNet project master. This option can be used to build the whole project yourself.
 
-Follow the instructions from https://mxnet.incubator.apache.org/install/osx_setup.html or https://mxnet.incubator.apache.org/install/ubuntu_setup.html
-about _Prepare Environment for GPU Installation_
-and _Install MXNet dependencies_
+### Getting Started with the Clojure Jars on Maven
```

Comment:
(###Getting-Started-with-the-Clojure-Jars-on-Maven) to create an anchor for this header.

Refer - [Github Markdown Heading Anchor](https://gist.github.com/asabaylus/3071099)